addon/nodes/block-rdfa.ts

```diff
@@ -9,6 +9,8 @@ export const block_rdfa: NodeSpec = {
     __tag: { default: 'div' },
   },
   defining: true,
+  isolating: true,
```

Comment:
unfortunately this also breaks current behavior
I suggest simply exporting a separate variation of this node, or just documenting how to extend nodes the host might want to treat like this.
we're in the early phases of the configuration interface where we simply expose all wires and ask the consumer to plug them in, so I'm ok with just documenting